shimmingtoolbox/__init__.py

```diff
@@ -6,3 +8,9 @@
     import importlib_metadata as metadata
 __version__ = metadata.version(__name__)
 del metadata
+
+__dir_shimmingtoolbox__ = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+__dir_testing__ = 'testing_data'
+__dir_config_dcm2bids__ = os.path.join(__dir_shimmingtoolbox__, 'config', 'dcm2bids.json')
```

Comment:
This is what https://setuptools.readthedocs.io/en/latest/pkg_resources.html is for.